packages/eslint-plugin-consent-manager/index.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
```

Comment:
relevant, our first eslint rule. 

This ensures people don't create integrations using `document.createElement('script')` to inject script tags. They should use our helper.